### PR TITLE
touch up Mizushima-san's abtract

### LINKED ISSUE #390: CFPの掲載@英訳: kmizu（水島宏太）さんによる「DOT計算をやさしく説明する」
```
---
name: kmizu (Kota Mizushima)
title: "Explain DOT calculus gently"
length: 40
audience: Advanced
language: Japanese
twitter: https://twitter.com/kmizu
github: https://github.com/kmizu
icon: https://pbs.twimg.com/profile_images/703094910417678337/b_v8jTJ3_400x400.jpg
organization: 株式会社ドワンゴ
tags:
  - Functional Programming, Others
suggestions:
  - * プログラミング言語に関する理論に興味がある人
* Scalaの理論的基盤に興味がある人
* より深くScalaを知りたい人
contributes:
  - 
speaker_experience:
  - 
---
The DOT calculus is the theory on which Dotty (which will be Scala 3.0) is based.  The DOT calculus is the model to understand the essence of Scala.  Such a model is so-called "core calculus" and it is used to prove characteristics of a programming language.  In this session, I'll explain about the DOT calculus gently.
```

日本語情報
タイトル - DOT計算をやさしく説明する
概要 - DOT計算とは、処理系Dotty（Scala 3.0になる予定です）の基礎となる理論です。DOT計算とは、Scalaの本質的な部分を抽出して、理論的に取扱いやすくしたモデルです。このようなモデルを核言語（core calculus）といい、プログラミング言語の性質を検証するために使われます。このセッションでは、DOT計算についてやさしく説明します。

- [ ] 英訳
- [ ] 英語版の掲載

### PR DESCRIPTION
**Pull request by eed3si9n**
Fixes #390

### CODE REVIEW

**Comment by taisukeoe** [comment]
LGTM

File: _candidates_en/kmizu_1.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+twitter: kmizu
+github: kmizu
+icon: https://pbs.twimg.com/profile_images/703094910417678337/b_v8jTJ3_400x400.jpg
+organization: Dwango, Inc
+tags:
+  - Functional Programming, Others
+suggestions:
+  - Those interested in programming language theory
+  - Those interested in the theoretical foundation of Scala
+  - Those who want to understand Scala deeoper
```
Comment (smroy) [inline _candidates_en/kmizu_1.md]: Line 16 Original:  - Those who want to understand Scala deeoper
Change: - Those who want to understand Scala deeper

File: _candidates_en/kmizu_1.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+contributes:
+  - fialyzer (https://github.com/dwango/fialyzer)
+  - Onion (https://github.com/onion-lang/onion)
+  - Klassic (https://github.com/klassic/klassic)
+speaker_experience:
+  - Scala Days 2010
+  - ScalaMatsuri 2018
+  - Scala Fukuoka 2019
+---
+The Dependent Object Types (DOT) calculus is the theory on which Dotty (which will be Scala 3.0) is based.  The DOT calculus is a model to understand the theoretical backbone of Scala. Such a model is called a "core calculus", and it is used to prove the characteristics of a programming language. In this session, I'll give a gentle introduction to the DOT calculus.
```
Comment (smroy) [inline _candidates_en/kmizu_1.md]: Line 26 Original: - The Dependent Object Types (DOT) calculus is the theory on which Dotty (which will be Scala 3.0) is based.  The DOT calculus is a model to understand the theoretical backbone of Scala. Such a model is called a "core calculus", and it is used to prove the characteristics of a programming language. In this session, I'll give a gentle introduction to the DOT calculus.
Change: - The Dependent Object Types (DOT) calculus is the theory on which Dotty (which will be Scala 3.0) is based. The DOT calculus is a model to understand the theoretical backbone of Scala. Such a model is called a "core calculus", and it is used to prove the characteristics of a programming language. In this session, I'll give a gentle introduction to the DOT calculus.

**Comment by eed3si9n** [comment]
@smroy Thanks Sandy. I'll send an additional pull request to reflect your change.

**Comment by eed3si9n** [comment]
Also this is the last abstract for this year 🎉

**Pull request merged**